crates/store/src/errors.rs

```diff
@@ -142,6 +142,8 @@ pub enum ApplyBlockError {
     DatabaseError(#[from] DatabaseError),
     #[error("I/O error: {0}")]
     IoError(#[from] io::Error),
+    #[error("Task join error: {0}")]
+    TokioJoinError(#[from] tokio::task::JoinError),
```

Comment:
Do we still need to add this error variant?

Reply:
Yes, this error might be thrown by `tokio::spawn` for `block_save_task` and `db_update_task`.